File: app/src/main/java/com/artmurka/artmurkaapp/Model/Pojo/ItemList/Checkout/OrderTotal.java

package com.artmurka.artmurkaapp.Model.Pojo.ItemList.Checkout;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class OrderTotal {

    @SerializedName("total")
    @Expose
    private String total;
    @SerializedName("total_raw")
    @Expose
    private Float totalRaw;

    public String getTotal() {
        return total;
    }

    public void setTotal(String total) {
        this.total = total;
    }

    public Float getTotalRaw() {
        return totalRaw;
    }

    public void setTotalRaw(Float totalRaw) {
        this.totalRaw = totalRaw;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/checkout/OrderTotal.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.checkout

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class OrderTotal {

    @SerializedName("total")
    @Expose
    var total: String? = null
    @SerializedName("total_raw")
    @Expose
    var totalRaw: Float? = null

}
